File: nicolaspuebla_proyecto_final/src/main/java/com/example/nicolaspuebla_proyecto_final/model/dto/LocalityListResponse.java
package com.example.nicolaspuebla_proyecto_final.model.dto;

import java.util.List;
import com.example.nicolaspuebla_proyecto_final.model.dataModels.Locality;

/**
 * Clase que representa la respuesta para la ista de localidades.
 */
public class LocalityListResponse {
    private List<Locality> localities;

    public LocalityListResponse() {}

    public LocalityListResponse(List<Locality> localities) {
        this.localities = localities;
    }

    public List<Locality> getLocalities() {
        return localities;
    }

    public void setLocalities(List<Locality> localities) {
        this.localities = localities;
    }
}

File: nicolaspuebla_proyecto_final_android/app/src/main/java/com/example/nicolaspuebla_proyecto_final_android/data/model/dto/LocalityListResponse.kt
package com.example.nicolaspuebla_proyecto_final_android.data.model.dto

import com.example.nicolaspuebla_proyecto_final_android.data.model.dataClases.Locality

/**
 * Clase que se utiliza para recibir una lista de localidades del microservicio.
 */
data class LocalityListResponse (
    val localities: List<Locality>
)